Engine/Commands/InvokeScriptAnalyzerCommand.cs

```diff
@@ -167,6 +167,18 @@ public SwitchParameter SuppressedOnly
         }
         private bool suppressedOnly;
 
+        /// <summary>
+        /// ShowAll: Show the suppressed and non-suppressed message
+        /// </summary>
+        [Parameter(Mandatory = false)]
+        [SuppressMessage("Microsoft.Performance", "CA1819:PropertiesShouldNotReturnArrays")]
+        public SwitchParameter IncludeSuppressions
+        {
+            get { return includeSuppressions; }
+            set { includeSuppressions = value; }
+        }
+        private bool includeSuppressions;
```

Comment:
```suggestion
        public SwitchParameter IncludeSuppressions { get; set; }
```

Reply:
I just followed the code style.

Reply:
In this case, the existing style is outdated and we should prefer the proposed style

Reply:
Do I need to revise other SwitchParameter to the proposed style? 
if I only write `includeSuppressions` to the proposed style it may look strange.

Reply:
For now we can just take the new format for the added code to keep this PR cohesive